app/src/main/java/com/github/studydistractor/sdp/distraction/Distraction.kt

```diff
@@ -16,7 +16,8 @@ data class Distraction(
     val long: Double? = null,
     val length : Length? = null,
     val tags: List<String>? = null,
-    val iconName: String? = null
+    val iconName: String? = null,
+    var distractionId: String? = null

```

Comment:
I prefer the "uid" name for this variable it is in the same format as in uid for the user

Reply:
Mhh I think it is still better to have explicit naming in dataclass, but we can put a format for naming data for next sprint.